mcx/distributions/mvnormal.py

```diff
@@ -0,0 +1,50 @@
+from jax import numpy as np
+from jax import random, scipy
+
+from mcx.distributions import constraints
+from mcx.distributions.distribution import Distribution
+from mcx.distributions.shapes import broadcast_batch_shape
+
+
+class MvNormal(Distribution):
+    params_constraints = {
+        "mu": constraints.real_vector,
+        "covariance_matrix": constraints.positive_definite,
+    }
+    support = constraints.real_vector
+
+    def __init__(self, mu, covariance_matrix):
+
+        (mu_event_shape,) = np.shape(mu)[-1:]
+        covariance_event_shape = np.shape(covariance_matrix)[-2:]
+        if (mu_event_shape, mu_event_shape) != covariance_event_shape:
+            raise ValueError(
+                (
+                    f"event shape {mu_event_shape} of mu does not match event "
+                    f"shape {covariance_event_shape} of covariance_matrix"
```

Comment:
I suspect that > 75% of users will run into shape errors with `MvNormal`. Maybe we should phrase it in terms of something they know (dimensions) instead of event shape (only devs do and should know about this):

"the number of dimensions {mu_event_shape} implied by the mean vector `mu` you provided "
"does not match the number of dimensions implied by the shape {covariance_event_shape} "
"of the covariance matrix you provided. "

What do you think?

Reply:
I like this. Ok will fix.